src/@types/Watchlist.tsx

```diff
@@ -0,0 +1,12 @@
+export type Watchlist = {
+  name: string
+  description: string
+  id: string
+  projects: number
+  createdAt: string
+}
+export enum WatchlistSorting {
+  RECENTLY_ADDED = `recentlyAdded`,
+  ALPHABETICAL = `alphabetical`,
+  NUMBER_OF_PROJECTS = `numberOfProjects`,
```

Comment:
Is there a reason for values to not match the casing of enum names?

Reply:
This was as the value is used in the translation so I used the same format that is used in the translation. Any preferences on how to handle this case?